platform/darwin/test/MGLDocumentationExampleTests.swift

```diff
@@ -47,6 +47,18 @@ class MGLDocumentationExampleTests: XCTestCase, MGLMapViewDelegate {
         styleLoadingExpectation.fulfill()
     }
     
+    func testMGLLight() {
+        //#-example-code
+        light = MGLLight()
```

Comment:
This needs to be declared as a new variable with `var` or `let` (hence the current test failures).

Reply:
🤦‍♂️  That's what I get for working on a Python project and a Swift project simultaneously right before a holiday. Thanks for taking a look, @friedbunny.

Reply:
In case it helps, this file is part of the test target, so you can build for test using ⇧⌘U to spot errors like this locally.